src/OVAL/probes/SEAP/sexp-manip.c

```diff
@@ -1421,7 +1424,13 @@ SEXP_t *SEXP_list_sort(SEXP_t *list, int(*compare)(const SEXP_t *, const SEXP_t
                            list_it_alloc, list_it_alloc + SEXP_LISTIT_ARRAY_INC);
 
                         list_it_alloc += SEXP_LISTIT_ARRAY_INC;
-			list_it = realloc(list_it, sizeof(SEXP_list_it) * list_it_alloc);
+                        void *new_list_it = realloc(list_it, sizeof(SEXP_list_it) * list_it_alloc);
```

Comment:
@evgenyz `new_list_it` probably should be assigned to `list_it`.

Reply:
Yes, it should. Thanks.